### PR TITLE
fix(clickhouse handler): add space between param and body when concat them

### PR DESCRIPTION
**Pull request by youngsofun**
I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

fix https://github.com/datafuselabs/databend/issues/8955#issuecomment-1333373881

### CODE REVIEW

**Review by TCeason — APPROVED** [review]

**Review by sundy-li — APPROVED** [review]

File: src/query/service/src/servers/http/clickhouse_handler.rs [hunk 1]
```diff
@@ -236,6 +236,7 @@ pub async fn clickhouse_handler_post(
 
     let default_format = get_default_format(&params, headers).map_err(BadRequest)?;
     let mut sql = params.query();
+    sql.push_str(" ");
```
Comment (sundy-li) [inline src/query/service/src/servers/http/clickhouse_handler.rs:239]: better add `if !sql.is_empty()`

Comment (youngsofun) [reply src/query/service/src/servers/http/clickhouse_handler.rs]: done

**Comment by andylokandy** [comment]
Seems that it break the CI...

**Comment by BohuTANG** [comment]
@youngsofun 
PTAL https://github.com/datafuselabs/databend/actions/runs/3597891674/jobs/6060124233#step:4:2485

**Pull request merged**